DataFormats/L1Trigger/interface/HOTPDigiTwinMux.h

```diff
@@ -0,0 +1,91 @@
+#ifndef HOTPDIGI_TWINMUX_H
```

Comment:
@rekovic - please follow the usual convention of <SUBSYSTEM>_<PACKAGE>_<FILE>_H in ifndefs here and elsewhere in this PR